### DISCUSSION TITLE
离线下载失败的文件怎么办？

### DISCUSSION THREAD

**Discussion by wuhuqifeiQWER** [AlistGo/alist#3106] [Q&A]
配合aria2使用了alist的离线下载功能，批量添加了很多文件，不过有很多文件在aria2下载失败了，如图
![image](https://user-images.githubusercontent.com/106081176/213393892-35231b67-843f-48f3-9a79-6f00b020ba87.png)
这些文件看起来没有被alist自动尝试重新下载，我也在aria2里尝试了重试这些下载任务，但是alist并没有上传这些文件，他们留在了alist的temp目录里，如何处理？怎么才能让alist自动重试aria2下载失败的文件？

**Comment by wuhuqifeiQWER** [comment]
或者有没有能手动让alist尝试重新上传temp目录里的文件的命令？

**Reply by ala4562** [reply]
我也遇到了报错

**Comment by wuhuqifeiQWER** [comment]
没人吗QAQ
